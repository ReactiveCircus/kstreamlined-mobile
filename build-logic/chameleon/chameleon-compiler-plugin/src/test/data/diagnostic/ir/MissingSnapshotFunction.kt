// SNAPSHOT_FUNCTION: /SnapshotTester.snapshot2
// THEME_VARIANT_ENUM: /ThemeVariant
// CHECK_COMPILER_OUTPUT
