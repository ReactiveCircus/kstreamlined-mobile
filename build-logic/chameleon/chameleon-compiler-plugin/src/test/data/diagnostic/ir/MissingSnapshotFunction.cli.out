error: could not find snapshot function </SnapshotTester.snapshot2>.
